modules/nerd-fonts/README.md

```diff
@@ -0,0 +1,20 @@
+# `nerd-fonts` installation module
```

Comment:
There are some issues in this file regarding and wording differences between it and the other modules. If this module expands, though, might need to make bigger changes, so don’t worry about this yet.